src/alt/index.js

```diff
@@ -77,6 +42,51 @@ class Alt {
     return makeAction(this, 'global', name, implementation, obj)
   }
 
+  createUnsavedStore(StoreModel, ...args) {
+    StoreModel.config = fn.assign({
+      invisible: true
+    }, StoreModel.config)
+    return this.createStore(StoreModel, null, ...args)
+  }
+
+  createStore(StoreModel, iden, ...args) {
+    // pull all prototype methods out
+    const proto = utils.getInternalMethods(StoreModel, true)
+
+    // force inheritance on store
+    utils.inherit(StoreModel, Store, Object.keys(proto).reduce((obj, name) => {
```

Comment:
Does this work if `StoreModel` extends another class? Looks like `getInternalMethods` might have to walk up the prototype chain for `StoreModel` for this to work.

I think it'd be easier just to pull out the `constructor` functionality on the base `Store` class into an `#init()` method, then invoke that (in both `Store#constructor` and for this), without messing with `StoreModel`'s prototype.